fuel-vm/src/interpreter/blockchain.rs

```diff
@@ -108,9 +108,11 @@ where
 
         // update frame pointer, if we have a stack frame (e.g. fp > 0)
         if fp > 0 {
-            let fpx = add_usize(fp, CallFrame::code_size_offset());
+            let fp_code_size = add_usize(fp, CallFrame::code_size_offset());
+            let fp_code_size_end = add_usize(fp_code_size, WORD_SIZE);
 
-            self.memory[fp..fpx].copy_from_slice(&length.to_be_bytes());
+            let length = length as Word;
```

Comment:
> Increment $fp->codesize, $ssp, and $sp by $rC padded to word alignment.
https://github.com/FuelLabs/fuel-specs/blob/master/src/vm/instruction_set.md#ldc-load-code-from-an-external-contract

Shouldn't this be added to the existing code size instead of replacing it with the length of code loaded?